pyfar/constants/constants.py

```diff
@@ -433,3 +434,149 @@ def density_of_air(
 
     # Equation 6.71
     return P / (R_a * (temperature + 273.15)) * (1+C)/(1+alpha*C)
+
+def fractional_octave_frequencies_nominal(num_fractions=1,
+                                          frequency_range=(20, 20e3)):
+    """Return the nominal center frequencies for octave-band and
+    one-third-octave-band filters.
+
+    Nominal center frequencies, as specified in the IEC 61260-1:2014 standard
+    [#]_, are standardized values that approximate the exact center
+    frequencies. In this function they are defined only for the range from
+    10 Hz to 20 kHz.
+
+    Parameters
+    ----------
+    num_fractions : 1, 3
+        The number of octave fractions. ``1`` returns octave center
+        frequencies, ``3`` returns third octave center frequencies.
+    frequency_range : array, tuple
+        The lower and upper frequency limits, the default is
+        ``(20, 20e3)``.
+        ``(20, 20e3)`` is in accordance with IEC 61260-1
+        ``(10, 20e3)`` is in accordance with IEC 61672-1 [#]_
```

Comment:
```suggestion
        The lower and upper frequency limits, the default is
        ``(20, 20e3)`` following IEC 61260-1 [#]_.
        E.g. ``(10, 20e3)`` would follow IEC 61672-1.
```
I hope this is correct.